person.rb

```diff
@@ -0,0 +1,40 @@
+class Person
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+    @id = Random.rand(1...5000)
+  end
+  # getter methods
+
+  def name_geter
+    @name
+  end
+
+  def id_getter
+    @id
+  end
+
+  def age_getter
+    @age
+  end
+  # setter methods
+
+  def name_setter(name)
+    @name = name
+  end
+
+  def age_setter(age)
+    @age = age
```

Comment:
- [ ] I think you don't longer require these methods after adding this line: `attr_accessor :name, :age` excluding Getter for `:id` which can equally be replace by` attr_reader :id`